src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,44 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
+
+  static deadAnimal() {
```

Comment:
The method name `deadAnimal` might be misleading as it suggests returning or handling a single dead animal. Consider renaming it to something like `removeDeadAnimals` to better reflect its functionality of filtering out dead animals from the `alive` array.